src/cli/commands/create.js

```diff
@@ -51,7 +52,10 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       throw new MessageError(reporter.lang('createInvalidBin', packageName));
     }
 
-    await child.spawn(binPath, rest, {stdio: `inherit`});
+    const linkedBinPath = config.getTemp(`create-bin-${packageName}`);
+    await linkBin(binPath, linkedBinPath);
```

Comment:
@arcanis Please review the new implementation. Tested on Windows.
Also I wonder, shouldn't we pass the flags to `runGlobal` (and now also to `getBinFolder`)?